app/models/ems_folder.rb

```diff
@@ -147,7 +148,7 @@ def register_host(host)
 
       host_mor                   = vim.computeResourcesByMor[cr_mor].host.first
       host.ems_ref               = host_mor
-      host.ems_ref_obj           = host_mor
+      host.ems_ref_type          = "HostSystem"
```

Comment:
Same, this whole method will move in https://github.com/ManageIQ/manageiq/pull/19534